bytecomp/symtable.ml

```diff
@@ -30,73 +32,84 @@ type error =
 
 exception Error of error
 
-let empty_numtable = { num_cnt = 0; num_tbl = Tbl.empty }
+module NumTbl (M : Map.S) = struct
```

Comment:
General point: I think the usual style is to have a single capital letter for module names and then lowercase afterwards separated by underscores.  (This also means that if you move a module out into a file of the appropriate name, then you can keep using lowercase filenames as is conventional, yet not have to rename any of the module references in the code.)

Reply:
Done.